charts/images.yaml

```diff
@@ -2,8 +2,8 @@ images:
 - name: etcd-backup-restore
   sourceRepository: github.com/gardener/etcd-backup-restore
   repository: eu.gcr.io/gardener-project/gardener/etcdbrctl
-  tag: "v0.14.0"
+  tag: "v0.15.0"
 - name: etcd
   sourceRepository: github.com/gardener/etcd-custom-image
   repository: eu.gcr.io/gardener-project/gardener/etcd
-  tag: "v3.4.13-bootstrap-1"
+  tag: "v3.4.13-bootstrap-2"
```

Comment:
Same as above

Reply:
Why? This value should be updated, right? We are not expecting this release of Druid to work with `v3.4.13-bootstrap-1` of `custom-image`

Reply:
Done